tools/mdanalysis/rdf.xml

```diff
@@ -24,15 +26,15 @@
 ]]></command>
     <inputs>
         <expand macro="analysis_inputs"/>
-        <param name="segid1"  type="text" value="PRO" label="Segid of atom 1"/>
-        <param name="resid1"  type="text" value="212" label="Resid of atom 1"/>
+        <param name="segid1"  type="text" value="PRO" label="Segment ID of atom 1"/>
+        <param name="resid1"  type="text" value="212" label="Residue ID of atom 1"/>
         <param name="name1"  type="text" value="OE2" label="Atom name of atom 1"/>
-        <param name="segid2"  type="text" value="HET" label="Segid of atom 2"/>
-        <param name="resid2"  type="text" value="3" label="Resid of atom 2"/>
+        <param name="segid2"  type="text" value="HET" label="Segment ID of atom 2"/>
+        <param name="resid2"  type="text" value="3" label="Residue ID of atom 2"/>
         <param name="name2"  type="text" value="C1" label="Atom name of atom 2"/>
         <param name="nbins"  type="text" value="100" label="Number of bins in the histogram"/>
-        <param name="start"  type="text" value="0.0" label="Starting Point"/>
-        <param name="end"  type="text" value="5.0" label="End Point"/>
+        <param name="start"  type="text" value="0.0" label="Starting point"/>
+        <param name="end"  type="text" value="5.0" label="End point"/>
```

Comment:
@tsenapathi @chrisbarnettster I really struggled to understand the purpose of these start/end params. Is it supposed to allow the user to just study a section of the trajectory?

Reply:
Yes. Either to remove initial section, test on first portion, etc. 
These are optional params though https://www.mdanalysis.org/docs/documentation_pages/analysis/rdf.html and the behaviour should probably be updated.

Reply:
But this kind of option could be useful for all tools, right? Not just RDF.

Reply:
yes definitely! I think by default should be 1 to the end but useful to allow this kind of functionality. 
@tsenapathi any feedback?

Reply:
Or even have a separate tool to cut slices out of trajectories?

Reply:
Separate tool could be mdconvert (part of mdtraj package). Supports striding and slice notation for frame selection.

Reply:
Cool, looks good. I'll make the tool then. Then we don't need to make an extra option for all the tools.

Reply:
Added a wrapper. Strangely, each time I slice a dcd file in the same way (on the command line) I get a different binary file, so had to use `sim_size` for the test. Not a problem for an xtc file.

Reply:
After rereading the mdanalysis documentation carefully I realized I misunderstood the nature of these parameters. They define a range of r over which the RDF is calculated. The slicing tool was still worth making though.